iOS/DuckDuckGo/AppLifecycle/AppStates/Foreground.swift

```diff
@@ -102,7 +102,7 @@ struct Foreground: ForegroundHandling {
             /// Handle **WebView related logic** here that could be affected by `AutoClear` feature.
             /// This is called when the **app is ready to handle web navigations** after all browser data has been cleared.
             onWebViewReadyForInteractions: {
-                vpnService.installRedditSessionWorkaround()
+                /* ... */
```

Comment:
hey @samsymons you can now move:
services.vpnService.resume() closer to other services' resume() [after interactionManager.start()]

Reply:
Oh this looks good... I was going to ask why the call above was deleted.  Makes sense.